rs/sns/governance/canister/governance.did

```diff
@@ -822,6 +870,26 @@ type GetUpgradeJournalResponse = record {
   upgrade_journal_entry_count: opt nat64;
 };
 
+type ListTopicsRequest = record {};
+
+type BuiltInAndGenericFunctions = record {
+  built_in_functions : vec nat64;
+  generic_functions : vec nat64;
+};
+
+type TopicInfo = record {
+  topic : Topic;
+  name : text;
+  description : text;
+  content : BuiltInAndGenericFunctions;
+  nested_topics : vec TopicInfo;
```

Comment:
Not sure we want this. Let's agree on the design level if we need nested topics.